scalafmt-tests/src/test/resources/newlines/source_keep.stat

```diff
@@ -3984,7 +3984,7 @@ object a {
 align.preset = none
 align.openParenDefnSite = true
 runner.dialect = scala3
-newlines.beforeOpenParenDefnSite = true
+newlines.beforeOpenParenDefnSite = source
```

Comment:
If the whole test is about keep, why do we set some of it to `source` and some to `keep`? Could we keep it all `source` or all `keep`? I think this brings in some unintended confusion.

Reply:
i wanted to test that both `source` and `keep` work. just minor variation in test cases, to prevent regression.